cores-and-flashing.tex

```diff
@@ -1,311 +1,254 @@
+\begin{center}
+  \fbox{\includegraphics[width=0.7\linewidth]{images/filehost_notsignedin.png}}
+\end{center}
 
-Click the {\bf Files} tab, and in the search-bar, type {\bf .cor} and press Enter.
-\ifdefined\printmanual
-  You will notice that there are different files with the {\tt .cor} extension. For your MEGA65, download the file
-  that ends with {\bf 65r3-dev.cor}. Other files are for other device types, which you can read more about in the
-  {\bf MEGA65 Book}.
-\else
-  For the purposes of this chapter on core-flashing, download the desired .cor file that suits your target device:
+To register a Filehost account with your owner code:
 
-  \begin{itemize}
-    \item{\textbf{mega65r3-dev.cor} (for MEGA65R3 boards, both Release and DevKits)}
-    \item{\textbf{mega65r2-dev.cor} (for MEGA65R2 boards)}
-    \item{\textbf{nexys4ddr-widget-dev.cor} (for Nexys4 DDR boards)}
-    \item{\textbf{nexys4-dev.cor} (for Nexys4 PSRAM boards)}
-    \item{You can also find .bit, .mcs and .prm files located here too.}
-  \end{itemize}
+\begin{enumerate}
+  \item Visit \href{https://files.mega65.org}{Filehost}. Click ``Sign Up.'' Follow the prompts to create an account.
+  \item Locate your owner code.\index{Owner Code} This is a code printed on a piece of paper that was included with your MEGA65 (possibly inserted into this manual). It looks something like this: {\tt 123-ABC-456}
+  \item Click the user icon in the upper-right corner of the Filehost screen. In the pop-up menu, select ``Redeem Code.'' Enter your owner code as prompted.
+\end{enumerate}
 
-  Alternatively, if you intend to flash the QSPI chip via Vivado, you would instead download the .mcs file for your target device (and optionally, the .prm files as well).
+\begin{center}
+  \fbox{\includegraphics[width=0.4\linewidth]{images/filehost_redeemmenu.png}}
+\end{center}
 
-  Another alternative for Nexys4 board users is to download .bit files and copy them to SD cards, which you can also download.
+To download the latest release package:
 
-  But once again, for the purposes of this chapter on core-flashing, you will only be interested in the .cor files.
-\fi
+\begin{enumerate}
+  \item Click the ``Files'' tab of the Filehost website.
+  \item In the search box on the left-hand side, type: ``release'' The list will update to show only files with that word in the title.
+  \item Locate the entry named, ``MEGA65 Core Release Package (mega65r3) incl. ROM,'' where ``mega65r3'' matches your hardware revision.
+  \item Click the entry. Confirm that this release package is for your hardware revision, then click ``Download'' to download the file.
+\end{enumerate}
+
+If you don't see an entry that says ``incl. ROM,'' check that you are signed in and that you have redeemed a valid owner code. Note that there is an entry for the Release Package that does not include the ROM that is visible to everyone. To ensure you are using a compatible set of files, get the package that says ``incl. ROM.''
```

Comment:
Disagree. For typable strings yes (and so also I wouldn't present typable strings inline with quote marks), but in this case it's a quoted phrase seen on a website, so for typographical reasons the ending punctuation goes inside the quotes. It's not ambiguous.

(I realize most people prefer "logical" punctuation to typographical punctuation these days, but IMO it's especially important for print typography.)

Reply:
Hmm, on this one, maybe I'll ask @Impakt's thought on how he feels about this.

For me, it was a little jarring, as for technical matters, if I see things in quotes, my eyes will look for those specific things within the quotes.

I had a read online on this topic:
- https://www.thesaurus.com/e/grammar/does-punctuation-go-inside-or-outside-quotation-marks/

A lot of the quote examples I see there seem to be of the spoken form, a quote of conversation of a character within a story, wanting to convey the slight pauses of that character's conversation.

So in that sense, yep, I believe I've seen punctuation used in that manner, though I don't feel like I've seen this pattern used in technical documentation.

Reply:
I agree that technical documentation should avoid typeable strings in regular quotes for this reason. Putting sentence punctuation outside of typographic quote marks is not a professional solution. The solution is to use a convention not based on quote marks. Casual and business comms are slipping towards `"blah",` but typesetters are not, because of the space below the quote mark and space above the sentence punctuation.

I used quotes in this case because it's not strictly a typeable string. It's English text, and it's not ambiguous. Nobody is going to see `"incl. ROM,"` in the text and `incl. ROM` on the website and think they are different. The alternative would be `{\bf incl. ROM}`. If it's really that important that this be considered a typeable string, then I can switch to that.